### PR TITLE
Conclude RDOC 2020

### PR DESCRIPTION
**Pull request by Pennek**

### CODE REVIEW

File: wiki/Tournaments/RDOC/2020/en.md [hunk 1]
```diff
@@ -27,35 +27,422 @@ The **Royal Danish osu!Cup 2020** (***RDOC 2020***) is an ongoing Danish 1v1 dou
...
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | ![][flag_DK] [iamVill](https://osu.ppy.sh/users/6295380) |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ![][flag_DK] [Pablohh](https://osu.ppy.sh/users/10092099) |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ![][flag_DK] [Vandabe](https://osu.ppy.sh/users/7050754) |
+
+## Mappools
+
+### Grand Finals
+
+- NoMod
+  - [hitorie - Little Cry Baby [Extreme]](https://osu.ppy.sh/beatmapsets/1098258#osu/2294643)
```
Comment (fajar13k) [inline wiki/Tournaments/RDOC/2020/en.md:113]:
```suggestion
  - [hitorie - Little Cry Baby \[Extreme\]](https://osu.ppy.sh/beatmapsets/1098258#osu/2294643)
```
You forgot to escape those brackets inside the difficulty?
Apply for others

File: wiki/Tournaments/RDOC/2020/en.md [hunk 2]
```diff
@@ -27,35 +27,422 @@ The **Royal Danish osu!Cup 2020** (***RDOC 2020***) is an ongoing Danish 1v1 dou
...
+  - [Kana Nishino - Sweet Dreams (11t dnb mix) [Andrea's Insane]](https://osu.ppy.sh/beatmapsets/499488#osu/1114962)
+- DoubleTime
+  - [O-Life Japan - Yamanobori Kibun [Lunatic]](https://osu.ppy.sh/beatmapsets/30256#osu/99989)
+  - [Nitro Fun - Cheat Codes VIP [Light Insane]](https://osu.ppy.sh/beatmapsets/711420#osu/1506487)
+
+## Match results
+
+### Grand Finals
+
+Sunday, 2020-09-27:
```
Comment (fajar13k) [inline wiki/Tournaments/RDOC/2020/en.md:283]:
```suggestion
Sunday, 2020-09-27:
```
Use this format instead, for clearer reading
```Sunday, 27 September 2020```
Also, no match links available?

**Review by fajar13k — COMMENTED** [review]
Also don't forget to update the Tournaments index too, and add the winners there

File: wiki/Tournaments/RDOC/2020/en.md (hunk 2 above)
Comment (Pennek) [reply wiki/Tournaments/RDOC/2020/en.md:283]: prefer keeping date consistent w/ other instances of the tournament

will add match links though

**Review by TPGPL — APPROVED** [review]

**Pull request merged**